test/integration/scheduler/preemption/preemption_test.go

```diff
@@ -430,7 +461,22 @@ func TestPreemption(t *testing.T) {
 			// Wait for preemption of pods and make sure the other ones are not preempted.
 			for i, p := range pods {
 				if _, found := test.preemptedPodIndexes[i]; found {
-					if err = wait.Poll(time.Second, wait.ForeverTestTimeout, podIsGettingEvicted(cs, p.Namespace, p.Name)); err != nil {
```

Comment:
Should we change `podIsGettingEvicted` instead?

Reply:
no need to, reverted the change and inspect the pod condition outside the poll as in other integration tests now